ecosystem/sep-0010.md

```diff
@@ -139,7 +139,81 @@ operation of the challenge is the **Home Domain**, and that the web auth domain
 GET <WEB_AUTH_ENDPOINT>
 ```
 
-Request Parameters:
+##### Request Header:
+
+Optionally, the server may forbid unauthorized calls to `GET <WEB_AUTH_ENDPOINT>`. This could be done to protect an
+endpoint against bad actors or limit access only to allowed applications.
+
+In that case, the client must add an `Authorization` header, and the server should respond to all requests with missing
+header with an error (see below). Server may choose to only accept calls from trusted applications, responding with an
+error to all other requests (see below).
+
+Authorized `WEB_AUTH_ENDPOINT` accepts `Authorization Bearer: <token>`, where token is [JSON Web Token](https://jwt.io/)
+authorizing the request.
+
+Client must specify the following claims:
+
+- `iat` (the time at which the JWT was issued
+  [RFC7519, Section 4.1.6](https://tools.ietf.org/html/rfc7519#section-4.1.6)) — current timestamp (for example
+  `1530644093`)
+- `exp` (the expiration time on or after which the JWT must not be accepted for processing,
+  [RFC7519, Section 4.1.4](https://tools.ietf.org/html/rfc7519#section-4.1.4))
+- `host` domain of the server to which the request is being made for.
+- `path` the path of the request (`<WEB_AUTH_ENDPOINT>`)
+- All request's parameters (if specified in the request)
```

Comment:
Just to clarify, you'd rather have fields in the JWT match the query parameters of the request? We use a different representation in the actual SEP-10 token and use the standardized `sub` field name.

```js
{
  "account": "G...",
  "memo": "12345123"
}
```
vs.
```js
{
  "sub": "G...:123456"
}
```

Reply:
Yes, I'd rather to keep it simple and just match query parameters (unlike SEP-10 which is issued and consumed by the same entity, this token is issued by the client application, so I'd rather to make it as obvious as possible)